### ISSUE TITLE
Gallery view doesn't change (Show/Hide) until resize the window

### ISSUE THREAD

**Issue by Chaniru22** [MarcAnt01/Fluent-Screen-Recorder#83]
https://drive.google.com/file/d/1YRx99YBmKjrqtFalF7rdz6onr5xgve-y/view?usp=sharing

**Comment by MarcAnt01** [comment]
Hey @Chaniru22 I am planning to remove the option since I think the app with a big window doesn't look good without it. What are your thoughts about this?

**Comment by Chaniru22** [comment]
@MarcAnt01 good idea . I like that .can't you disable the controls until record finished ?

**Comment by Chaniru22** [comment]
@MarcAnt01 yeah .I like it .
